### ISSUE TITLE
[CAIA A11y Research]: Debt Resolution - Debt Portal: Payment History (July 2024)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#88124]
> [!NOTE]
> **High-Level Overview**: Debt Resolution
> - **Research Timeframe**: 7/16 - 7/23/2024 (7/17 and 7/18/2024)
> - **Time of Day:** Afternoon
> - **Pilot**:  7/16/2024` #87817 
> - **Assistive Tech Users**: `3` AT Users. (available coverage 1  pilot and 1 session)
> - **Artifacts Available**: 7/8/2024
> - **Testing Environment:** [Prototype in CodePen](https://codepen.io/team/Debt-Team/live/rNgqxLY)
> - **Assistive Tech Being Used**: `VoiceOver` `Screen magnifier`
> - **Task Lists**: See task lists for next steps/definition of done

## CAIA Assignee(s)

<details><summary>Toggle to view information about the CAIA Team.</summary>

- Sara
- @EvanAtCoforma 
- @rileyorr 
- See note re: limited team coverage w/o 7/15/2024

</details>

## About the Team

<details><summary>Toggle to view information about the team.</summary>

- Team name: Debt Resolution
- OCTO product owner: Denise Coveyduc
- Product name: Debt Portal
- Product manager: Tom Davis
- Moderators: Charlotte Cesana and Megan Gayle
- Note-takers: Megan Gayle, Joseph Lee, Perigean
- Slack channel: [#debt-resolution](https://join.slack.com/share/enQtNzM5MTE3ODIwMDQ4My1hYTY3OGIyZGYzMGYwZjkyYWYyMzg5OTE4MDhiODRiOTU5ZGQ3ZGIzOTMyNTRjMzcwMjA4ZGZkZTE5MjNjY2Fm)
- Dedicated content writer on your team (if you have one): NA
- Dedicated a11y specialist on your team (if you have one): NA

</details>

<details><summary>Toggle to view more intake research ticket information</summary>

### Do you have an accessibility champion?

No, we don't have a champion (yet)

### Calendar checking

- [ ] Based on this calendar, confirm we have specialists who can support your study in your desired timeframe
- [ ] Confirm you are filing this support ticket at least 1 week in advance of said timeframe, so we have enough heads up to provide support

### Have you reviewed the CAIA capacity calendar?

- [ ] Yes, we have reviewed the CAIA capacity calendar and can confirm specialists are available to support in our research timeframe

### When is the timeframe for your testing?

- Pilot Requested: Week of July 15, on July 16, 2024 (Tuesday)
- Research: Tuesday, July 16 , 2024 - Tuesday, July 23 , 2024

[Per the 7/15/2024 pilot ticket comment ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/87817#issuecomment-2228866446)

> Here are the times/dates when participants who are using screen readers will be joining:
> - P3- Wednesday, July 17 3:00 – 4:00pm
> - P4- Thursday, July 18 1:00 – 2:00pm
> - P5- Thursday, July 18 4:00 – 5:00pm

</details>

## Research Sessions

> [!TIP]
> - Providing CAIA A11ys with the time slots in advance for AT users, will allow a11ys to avoid conflicts on the CAIA A11y Capacity Calendar, and reach out to other a11ys, should CAIA not be available. 
> - Providing CAIA A11ys with the technology and devices the users will be testing with, will allow CAIA to prepare in advance for the sessions. 

<details><summary>Toggle to view research timing information.</summary>

### Pilot Timeframe
- **Pilot Session**:`Requested`
- **Date Requested for PIlot**: 7/16/2024 (req. 7/8/2024 via ticket #87817
- [Reference Comment c/o Megan Gayle on 7/8/2024](https://github.com/department-of-veterans-affairs/va.gov-team/issues/87817#issuecomment-2217997121)
> We are hoping to begin the study next week on the 16th. Would someone be available for a Pilot session before then?
> Additionally, can someone join sessions with participants that utilize screen readers?

### Research Timeframe
- **[Research Dates (per research plan)](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/combined_va_debt_portal/payment-history/research/usability-study-6-2024/research-plan.md#availability)**
    - Start of Research: Tuesday, July 16 , 2024
    - End of Research: Tuesday, July 23 , 2024

### Length of sessions

- **Session length:** `60` minutes
- **Buffer time between sessions:** `60` minutes
- **Maximum sessions per day:** `2`

</details>

###  Testing Schedule

> [!CAUTION]
> #### LIMITED CAPACITY JULY 2024 NOTE
> - There will be limited CAIA A11Y capacity the week of 7/15. 
> - There are currently multiple studies the week of 7/15, two studies have scheduled sessions on the CAIA Capacity Calendar and two studies have holds. 
> - Tuesday, July 16; however, we should have time to run a pilot. 
> - @mgleo and @charlottecesana Just let us know as soon as you have your two SR sessions scheduled and we can see if we can cover those as well. Thanks so much! 

CAIA A11Ys will be looking for device and AT information. We realize that some of this information will not be available until the day of the session. 

Please toggle and use the following table to enter the following information for each participant: Date and Time, Assistive tech (AT), Device and Operating System, Browser, and Skill level:

<details><summary>Toggle to view the AT testing schedule</summary>

P# | Date and Time (ET) | AT | Device and OS | Browser | Skill | CAIA A11ys Available
------------------|------------------|--------------|--------------|--------------|--------------|--------------
Pilot Session | 7/16/24, `Time TBD` | `Screen Reader - needs specified by Team` VoiceOver, JAWS, etc `TBD` | Mobile or Desktop; Windows, MacOS, iOS, Android, etc. `TBD` | Safari, Chrome, etc. `TBD` | `NA-Pilot`  | @EvanAtCoforma 
P3 | 7/17/2024, 3- 4 p.m. | VoiceOver, JAWS, etc `TBD` | Mobile or Desktop; Windows, MacOS, iOS, Android, etc. `TBD` | Safari, Chrome, etc. `TBD` | Novice, Intermediate, Advanced `TBD` | @sara-amanda
P4 | 7/18/2024, 1-2 p.m. | VoiceOver, JAWS, etc `TBD` | Mobile or Desktop; Windows, MacOS, iOS, Android, etc. `TBD` | Safari, Chrome, etc. `TBD` | Novice, Intermediate, Advanced `TBD` | **Unable to cover**
P5 | 7/18/2024, 4-5 p.m. | VoiceOver, JAWS, etc `TBD` | Mobile or Desktop; Windows, MacOS, iOS, Android, etc. `TBD` | Safari, Chrome, etc. `TBD` | Novice, Intermediate, Advanced `TBD` | **Unable to cover**

- **Consideration**: You can also consider asking participants more specific AT and device combinations. 
- **For instance:** TalkBack on Samsung Galaxy 8, or VoiceOver on iPad with magnification.

</details>

CAIA A11ys will review the coverage needed, and self-assign, based on a11y availability.

## Supporting artifacts 

<details><summary>Toggle to view the supporting artifacts</summary>

- [Prototype in CodePen](https://codepen.io/team/Debt-Team/live/rNgqxLY)
- [Research Convo Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/combined_va_debt_portal/payment-history/research/usability-study-6-2024/at-conversation-guide.md)
- [Research Plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/combined_va_debt_portal/payment-history/research/usability-study-6-2024/research-plan.md)

</details>

## Next Steps: Definition of Done

```[tasklist]
### CAIA A11Y Tasks
- [x] **INTAKE TICKET:** @sara-amanda processing intake ticket - internal ticket template for pilots was used instead of the research request ticket. Grabbed ticket number via Slack (thanks for posting in our channel, sorry for the delay it didn't show in GHP due to template used.) Updating and creating new research intake ticket and refining to address requests. 👍 We are good!
- [x] **PILOT SESSION: REQUESTED** Pilot session with team. `Requesting 7/16/2024`
- [x] **PILOT SESSION: CREATE TICKET** #87817
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/87817
- [x] **PILOT SESSION: SCHEDULE** Add to calendar
- [x] **CALENDAR HOLDS** Update A11y [CAIA Capacity Calendar](https://calendar.google.com/calendar/u/0/embed?src=c_f982350fc73a8162a1088ab14d00dd37335b2b559a60d6b67eb8021416f81c2a@group.calendar.google.com&ctz=America/Chicago) with holds for user research sessions with AT, when dates become available.
- [x] **REMOVE CALENDAR HOLDS** Remove holds on A11y [**CAIA Capacity Calendar**](https://calendar.google.com/calendar/u/0/embed?src=c_f982350fc73a8162a1088ab14d00dd37335b2b559a60d6b67eb8021416f81c2a@group.calendar.google.com&ctz=America/Chicago) and replace with invitations from project team (or Perigean)
- [x] **RESEARCH NOTES** Create CAIA notes document
- [x] **Divvy user research session support with CAIA A11ys** - based on availability and technology available to support. Limited availability due to PTO and other research sessions. Unable to cover more than one session plus pilot for the total coverage of 2 sessions.
- [x] **Review research folder and assets**
- [x] **Participate in research** **Date Span:**`7/17/2024` and `7/18/2024` - 3 sessions (as of 7/15/2024) - pilot 7/16 and 1 session on 7/17 only - due to PTO and other research session conflicts.
- [x] **SESSION NOTES**: Create [CAIA notes document](https://docs.google.com/document/d/1Y9PYcsDGK95T25ct8dgHiN-wGI0suF-fDMWne2xGKtg/edit) for a11ys to use during the sessions
- [x] **SYNC:** Connect with CAIA A11ys on research findings (meeting)
- [ ] **Create/update the CAIA A11y Research Deliverable** This is dependent on the number of study participants CAIA supports. If minimal, A11Ys may share feedback only in the feedback Mural or this ticket.
- [ ] **Share the completed deliverable** with the project team via Slack: `Primary Team Slack` and `Ticket`
```

```[tasklist]
### VFS TEAM Tasks
- [x] Tag @Naomi Eke in Slack [#sitewide-content-accessibility-ia](https://dsva.slack.com/archives/C01K37HRUAH)
- [ ] Your team's researchers and observers read through platform guidance on research with assistive technology users
- [ ] Review our [assistive technology pilot guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/assistive-tech-pilot-guide.md) and request a pilot, if desired
- [x] Request a pilot session with CAIA a11ys #87817 `Team is Requesting 7/16/2024`
- [x] Schedule a pilot session with CAIA a11ys
- [ ] (OPTIONAL) Receive feedback on Conversation Guide for Screen Readers (if requested)
- [ ] **PROTOTYPE:** Provide prototype to CAIA A11ys via this ticket in the supporting artifacts section
- [x] **DATES NEEDED** See `Table` in the ticket above. Please provide date range and slots for AT sessions being planned.
- [ ] **ASSISTIVE TECH. INFO NEEDED:** Please provide the AT the users will be utilizing during the research sessions in `Table` in the ticket above.
- [ ] **INVITES:** Forward invites from Perigean to CAIA A11ys (Sara or Naomi)
```

**Comment by sara-amanda** [comment]
## Sessions 

[Per the 7/15/2024 pilot ticket comment:](https://github.com/department-of-veterans-affairs/va.gov-team/issues/87817#issuecomment-2228866446)

### Limited Coverage
**Also see table in this ticket body:** 
- [x] Pilot - Tuesday, July 16 at 1:30 p.m. - `Confirmed & Covered` @EvanAtCoforma 
- [x] P3- Wednesday, July 17 3:00 – 4:00pm - `Covered` @sara-amanda 
- [ ] P4- Thursday, July 18 1:00 – 2:00pm - `Unable to cover`
- [ ] P5- Thursday, July 18 4:00 – 5:00pm - `Unable to cover`

@mgleo and @charlottecesana - we are so sorry we are 99.9% unable to cover those last two sessions, due to our limited availability this week. We can; however, cover the pilot, which should prove to be very beneficial and the one session on July 17. We hope this still helps you, our sincere apologies. 

cc: @EvanAtCoforma @coforma-jamie @NaomiPMC

**Comment by mgleo** [comment]
@sara-amanda Thank you for the update and for covering the session tomorrow. The session tomorrow will be with a participant who is using VoiceOver. 

@EvanAtCoforma, thank you for helping us with the pilot. Looking forward to speaking with you today.


In case anything changes, the first session on Thurs is with a participant using JAWS and the second session is with a participant using VoiceOver.

**Comment by sara-amanda** [comment]
@mgleo @charlottecesana I am not seeing a link for the session yet can you point us to it? Thanks so much! 😄
